client/src/components/icons/LogoutIcon.js

```diff
@@ -13,17 +15,22 @@ const Logout = styled.path`
   :hover {
     fill: ${props => props.theme.colors.secondary};
   }
+  :active {
+    fill: ${props => props.theme.colors.primary};
+  }
 `;
 
 export default function LogoutIcon() {
   return (
-    <Svg
-      xmlns="http://www.w3.org/2000/svg"
-      width="36"
-      height="31.5"
-      viewBox="0 0 36 31.5"
-    >
-      <Logout d="M13.5,0V4.5h18V27h-18v4.5H36V0ZM9,9,0,15.75,9,22.5V18H27V13.5H9Z" />
-    </Svg>
+    <Button to="/">
+      <Svg
+        xmlns="http://www.w3.org/2000/svg"
+        width="36"
+        height="31.5"
+        viewBox="0 0 36 31.5"
```

Comment:
... {...props} here. Furthermore you can delete width and height here as you have it in line 9 & 10.